src/charm.py

```diff
@@ -672,6 +672,9 @@ def _reconcile_runners(self, runner_manager: RunnerManager) -> Dict[str, Any]:
         delta_virtual_machines = runner_manager.reconcile(
             virtual_machines, virtual_machines_resources
         )
+
+        self.unit.status = ActiveStatus()
+
```

Comment:
Should this be set by the controller function that calls `_reconcile_runners`?

Reply:
This function was already setting the MaintenanceStatus at line 664.
This change is to ensure status is set back to active.

When creating this function I was debating whether the controller function(top-level charm event handler functions) should be the one setting the status. However, this results in three place of the code having something like:
```
self.unit.status = MaintenanceStatus("Reconciling")
self._reconcile_runners()
self.unit.status = ActiveStatus()
```
So I decided that it might be better if helper function within `charm.py` was allow to set the status as well.